lib/date-utils.js

```diff
@@ -141,10 +141,3 @@ export const convertDateToApiUtc = (date, timezone) => {
 export const convertDateFromApiUtc = (date, timezone) => {
   return dayjs(date).tz(timezone).format('YYYY-MM-DD HH:mm:ss');
 };
-
-/**
- * Check if the date is valid while editing
- */
-export const isValidDate = date => {
-  return date && !date.startsWith('0');
-};
```

Comment:
Replaced this by a simple Boolean check as a date like `0008-01-01` stays a valid date